api_app/analyzers_manager/file_analyzers/cape_sandbox.py

```diff
@@ -0,0 +1,69 @@
+import time
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class CAPEsandbox(FileAnalyzer):
+    def set_params(self, params):
+        self.__token = self._secrets["token"]
+        self.__vm_name = params.get("VM_NAME", "win7x64_8")
+
+    def run(self):
+        API_URL: str = "https://www.capesandbox.com/apiv2/tasks/create/file/"
+        headers = {
+            "Authorization": f"Token {self.__token}",
+        }
+
+        files = {
+            "file": (self.filename, self.read_file_bytes()),
+        }
+
+        data = {"machine": self.__vm_name}
+
+        response = requests.post(API_URL, headers=headers, files=files, data=data)
+        response_json = response.json()
+        if not response_json["error"]:
+            to_respond = {}
+            to_respond["result_url"] = response_json["url"]
+            task_id = response_json["data"]["task_ids"][0]
+            status_api = "https://www.capesandbox.com/apiv2/tasks/status/" + str(
+                task_id
+            )
+            while True:
+                r = requests.get(status_api, headers=headers)
+                responded_json = r.json()
+                error = responded_json["error"]
+                data = responded_json["data"]
+                if (not error) and (data == "reported" or data == "completed"):
```

Comment:
parenthesis not required for the first condition. Second condition should use `in`

Reply:
Why `in` matteo?

Reply:
As in, It always returns the whole string "reported" or "completed" accordingly. I usually use `in` only when i need to check if one string co-insides with another. I will definitely add it as well but I wanted to understand why did you feel the need for it?

Reply:
ahh i see what you meant! https://deepsource.io/gh/intelowlproject/IntelOwl/run/55667e73-a3d2-4e5c-94ee-b8d92e1f4da6/python/PYL-R1714

Reply:
yeah coming late to your questions but I see you solved your questions by yourself. :P That's appreciated.